test/Dapr.E2E.Test.App.Grpc/MessageRepository.cs

```diff
@@ -0,0 +1,49 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation and Dapr Contributors.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Dapr.E2E.Test
+{
+    public class MessageRepository
+    {
+        private readonly IDictionary<string, Queue<string>> messages;
+        private readonly Queue<string> broadcasts;
+
+        public MessageRepository()
+        {
+            this.messages = new Dictionary<string, Queue<string>>();
+            this.broadcasts = new Queue<string>();
+        }
+
+        public void AddMessage(string recipient, string message)
+        {
+            if (!this.messages.ContainsKey(recipient))
+            {
+                this.messages.Add(recipient, new Queue<string>());
+            }
+            this.messages[recipient].Enqueue(message);
+        }
+
+        public string GetMessage(string recipient)
+        {
+            if (this.messages.ContainsKey(recipient) && this.messages[recipient].Peek() != null)
+            {
+                return this.messages[recipient].Dequeue();
+            }
+            return string.Empty;
+        }
+
+        public void AddBroadcast(string message)
+        {
+            this.broadcasts.Enqueue(message);
+        }
+
+        public string GetBroadcast()
+        {
+            return this.broadcasts.Dequeue();
+        }
```

Comment:
AddBroadcast and GetBroadcast are unused

Reply:
A relic from before I realized I could stream in and out of the same method.